examples/otel-logs-splunk-raw-hec/README.md

```diff
@@ -0,0 +1,31 @@
+# Splunk Raw HEC example
+
+This example showcases how the collector can expose a HTTP endpoint to receive data over the HEC endpoint as a raw payload.
+
```

Comment:
Suggested change: "This example shows how the Collector can expose an HTTP endpoint to receive data over the Splunk HTTP Event Collector (HEC) endpoint as a raw payload."